GeorgianMemenator/Scenes/DogBreedDetail/DogBreedDetailView.swift

```diff
@@ -14,11 +14,23 @@ struct DogBreedDetailView: View {
         VStack {
             if viewModel.hasError {
                 Text("Error")
-                    .font(.title)
-                    .padding(.top, 32)
             } else if viewModel.isLoading {
                 ActivityIndicator(isAnimating: true)
+            } else {
+                VStack(alignment: .leading, spacing: 4) {
```

Comment:
Lets align entire details to top of the screen, currently its in the middle.
Also please, add paddings from left and right of 16 pixels.